CHANGELOG.rst

```diff
@@ -2,14 +2,192 @@
+  you specified this as ``example.com,cms.example.com``, then the value needs to be
+  updated to ``https://example.com,https://cms.example.com``.
+
+  Check (and update) your infrastructure code/configuration for this setting before
+  deploying.
+
+* The Objects API registration backend can now update the payment status after
+  registering an object. For this feature to work, the minimum version of the Objects
+  API is now ``v2.2`` (raised from ``v2.0``). If you don't make use of payments or don't
+  store payment information in the object, you can likely keep using older versions, but
+  this is at your own risk.
+
+* The ``TWO_FACTOR_FORCE_OTP_ADMIN`` and ``TWO_FACTOR_PATCH_ADMIN`` environment variables
+  are removed, you can remove them from your infrastructure configuration. Disabling MFA
+  in the admin is no longer possible. Note that the OpenID Connect login backends do not
+  require (additional) MFA in the admin and we've added support for hardware tokens
+  (like the YubiKey) which make MFA less of a nuisance.
+
+Major features
+--------------
+
+**📄 Objects API contract**
+
+We completely revamped our Objects API registration backend - there is now tight
+integration with the "contract" imposed by the selected object type. This makes it
+much more user friendly to map form variables to properties defined in the object type.
+
+The existing template-based approach is still available, giving you plenty of time to
+convert existing forms. It is not scheduled for removal yet.
+
+**👔 Decision engine (DMN) support**
+
+At times, form logic can become very complex to capture all the business needs. We've
+added support for evaluation of "Decision models" defined in a decision evaluation
+engine, such as Camunda DMN. This provides a better user experience for the people
+modelling the decisions, centralizes the definitions and gives more control to the
+business, all while simplifying the form logic configuration.
+
+Currently only Camunda 7 is supported, and using this feature requires you to have
+access to a Camunda instance in your infrastructure.
+
+**🔑 Multi-factor rework**
+
+We've improved the login flow for staff users by making it more secure *and* removing
+friction:
+
+* users of OIDC authentication never have to provide a second factor in Open Forms
+* you can now set up an automatic redirect to the OIDC-provider, saving a couple of
+  clicks
+* users logging in with username/password can now use hardware tokens (like YubiKey),
+  as an alternative one-time-password tokens (via apps like Google/Microsoft
+  Authenticator)
+
+**🔓 Added explicit, public API endpoints**
+
+We've explicitly divided up our API into public and private parts, and this is reflected
+in the URLs. Public API endpoints can be used by CMS integrations to present lists of
+available forms, for example. Public API endpoints are subject to semantic versioning,
+i.e. we will not introduce breaking changes within a major version.
```

Comment:
I think you meant that they will only introduce breaking changes in major versions? Or that they won't introduce breaking changes in minor versions?
Or maybe I misunderstand :upside_down_face:

Reply:
reworded!